resources/lang/fr/dashboard.php

```diff
@@ -1,11 +1,11 @@
 <?php
 
 return [
-    'totalhours'     => 'Heures Totales',
-    'yourbalance'    => 'Votre Solde',
-    'yourlastreport' => 'Votre dernier Rapport',
-    'noreportsyet'   => 'Aucun Rapport',
+    'totalhours'     => 'Total heures',
+    'yourbalance'    => 'Votre solde',
+    'yourlastreport' => 'Votre dernier rapport',
+    'noreportsyet'   => 'Pas de apport déposé',
```

Comment:
There is a missing "r" it's "Pas de rapport déposé"

Reply:
Well seen ! 
Tipo

Reply:
Jje corrige